CHANGELOG.md

```diff
@@ -1329,7 +1344,9 @@ Install with `npm install onfido-sdk-ui@0.12.0-rc.1`
 
 - NPM (commonjs2) style of importing the library now works
 
-[next-version]: https://github.com/onfido/onfido-sdk-ui/compare/9.0.0...development
+[next-version]: https://github.com/onfido/onfido-sdk-ui/compare/null...development
+[null]: https://github.com/onfido/onfido-sdk-ui/compare/9.1.0...null
+[9.1.0]: https://github.com/onfido/onfido-sdk-ui/compare/9.0.0...9.1.0
```

Comment:
And this as well

Reply:
Looks like something went wrong indeed. For this release it's fixed now